<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="KAMILA-PC" name="testNGFramework.Assertions" tests="7" failures="1" timestamp="20 Aug 2020 02:56:54 GMT" time="25.162" errors="0">
  <testcase name="c_assertNotEquals" time="0.102" classname="testNGFramework.Assertions"/>
  <testcase name="g_assertNull" time="0.004" classname="testNGFramework.Assertions"/>
  <testcase name="a_pageLoad" time="24.636" classname="testNGFramework.Assertions"/>
  <testcase name="b_assertequals" time="0.172" classname="testNGFramework.Assertions">
    <failure type="java.lang.AssertionError" message="expected [ONLINE BANKINGG] but found [ONLINE BANKING]">
      <![CDATA[java.lang.AssertionError: expected [ONLINE BANKINGG] but found [ONLINE BANKING]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at testNGFramework.Assertions.b_assertequals(Assertions.java:23)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- b_assertequals -->
  <testcase name="f_assertNotNull" time="0.032" classname="testNGFramework.Assertions"/>
  <testcase name="d_assertTrue" time="0.090" classname="testNGFramework.Assertions"/>
  <testcase name="e_assertfalse" time="0.126" classname="testNGFramework.Assertions"/>
</testsuite> <!-- testNGFramework.Assertions -->
